Sources/ConcordiumSwiftSDK/Model/Error.swift

```diff
@@ -1,5 +1,15 @@
 import Foundation
 
+infix operator ?!: NilCoalescingPrecedence
+
+/// Throws the `Error` on the right hand side  if the left hand side is `nil`.
```

Comment:
Not all developers might be able to decipher this method signature (operator overloading, generics and autoclosure combined :) - so I would suggest adding an example to the doc comment to make it 100% clear how to use it.

Reply:
I can't get the comment to actually render in XCode, but I extended it with an example.